### PR TITLE
docs: fix format issues

### PR DESCRIPTION
**Pull request by yizha1**
Updates:
- Fix format issues

Signed-off-by: Yi Zha <yizha1@microsoft.com>

### CODE REVIEW

**Comment by yizha1** [comment]
@shahramk64 Please help to review the changes since you tried testing with this doc.

File: versioned_docs/version-1.2/quickstarts/ratify-on-azure.md [hunk 1]
```diff
@@ -449,13 +449,13 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
     apiVersion: config.ratify.deislabs.io/v1beta1
     kind: KeyManagementProvider
     metadata:
-    name: keymanagementprovider-akv
+      name: keymanagementprovider-akv
```
Comment (shahramk64) [inline versioned_docs/version-1.2/quickstarts/ratify-on-azure.md:452]: This needs to be reflected in the following kubectl commands as well:
KeyManagementProvider-akv --> keymanagementprovider-akv

File: versioned_docs/version-1.2/quickstarts/ratify-on-azure.md [hunk 2]
```diff
@@ -491,26 +491,26 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
...
-    parameters:
+      name: notation
+      artifactTypes: application/vnd.cncf.notary.signature
+      parameters:
         verificationCertStores:
         certs:
-            - keymanagementprovider-akv
+          - keymanagementprovider-akv
         trustPolicyDoc:
         version: "1.0"
```
Comment (shahramk64) [inline versioned_docs/version-1.2/quickstarts/ratify-on-azure.md:503]: Needs to be indented

File: versioned_docs/version-1.2/quickstarts/ratify-on-azure.md [hunk 3]
```diff
@@ -491,26 +491,26 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
...
         trustPolicies:
```
Comment (shahramk64) [inline versioned_docs/version-1.2/quickstarts/ratify-on-azure.md:504]: ditto

File: versioned_docs/version-1.2/quickstarts/ratify-on-azure.md [hunk 4]
```diff
@@ -491,26 +491,26 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
...
-            - name: default
```
Comment (shahramk64) [inline versioned_docs/version-1.2/quickstarts/ratify-on-azure.md:505]: The rest of this yaml file needs to be indented properly once the above two indentations are corrected.

**Review by shahramk64 — CHANGES_REQUESTED** [review]
I had a similar PR a while ago and went through most of this to make it work for version 1.3. You can refer to it here:
https://github.com/ratify-project/ratify-web/pull/106/files

**Comment by yizha1** [comment]
Thanks @shahramk64 please review it again. I added two files which had similar issues.

**Review by shahramk64 — APPROVED** [review]
LGTM

**Review by junczhu — APPROVED** [review]
LGTM

**Review by susanshi — APPROVED** [review]
LGTM. thanks!

**Pull request merged**